.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,5 +22,4 @@ Closes # <-- _insert Issue number if one exists_
 - [ ] added/updated copyright headers?
 - [ ] documented public classes/methods?
 - [ ] added/updated relevant documentation?
-- [ ] added relevant details to the changelog? (_skip with label `no-changelog`_)
```

Comment:
instead of completely removing this section I would add a bullet point:
`- [ ] assigned correct labels (exclude from changelog with label "no-changelog")?`
 
A bit of background: due to the auto-generating of Changelogs, PRs must now have the correct labels as well, so that they get categorized correctly. If no label is assigned, the PR will get categorized under `Features & Improvements`. `bug` and `documentation` should be self-explanatory.

Reply:
Added your suggestion into the template.

Reply:
@paullatzelsperger Maybe self-explanatory but now known that this will be filtered. Do we have an explanation anywhere? Otherwise, I would rephrase to:
```
- [ ] assigned correct labels? (`bug`, `documentation`, exclude from changelog with `no-changelog`)
```

Reply:
I think so, but using one of those 3 labels is not mandatory. Every other PR will also end up in the changelog. Does it make sense to force people to use at least one label? @ndr-brt

Reply:
@juliapampus my only concern is to "forget" to set the label and have a lot of PRs going to the wrong section by default (I tend to forget about checklist after some PRs). Having it mandatory will make everyone to have to think about what's the right section, but I agree on the fact that could be bothersome.

Reply:
@ndr-brt @paullatzelsperger Would something like #1737 help?

Reply:
That would also allow to filter the automated changelog in more detail